posts/marker-gene-smoothing/index.qmd

```diff
@@ -0,0 +1,627 @@
+---
+title: "Applications for visualization and cell typing using 'smoothed' marker genes" 
+author:
+  - name: Dan McGuire
+    orcid: 0009-0006-0286-9625
+    affiliations:
+      - ref: nstg
+      - ref: dan11mcguire
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 4
+date: "2024-06-07"
+date-modified: last-modified
+categories: [visualization, cell typing]
+image: figures/plot_oneCosMxFOV-1.png
+description: "Applications for visualization and cell typing using 'smoothed' marker genes based on expression nearest neighbors"
+draft: false
+format:
+  html: 
+    code-tools: true
+---
+
+# Introduction 
```

Comment:
This is a nice and thorough discussion. I think what's missing is quick tips. E.g., "how do I subset T-cells with smoothed marker genes?" (Take all insitutype T-cells, then cluster or gate using smoothed markers.)

Maybe add a bulleted list at the end of the intro, something like the below:

"In this post, you'll see:
- Why counts of a single marker gene are not definitive of cell type
- How to derive more useful / less noisy / "smoothed" expression of marker genes
- How to perform fine-grained subtyping using smoothed marker genes, with T-cell subtyping as motivation"